app.config.js

```diff
@@ -0,0 +1,8 @@
+export default ({ config }) => {
+  return {
+    ...config,
+    extra: {
+      appBuildVersion: process.env.APP_BUILD_VERSION
+    }
+  };
+};
```

Comment:
Is it possible to have it as `app.config.js`, so we minimise the use of `.js` files

Reply:
erm you mean `.ts`?

Reply:
yep, sorry typo. If the typescript types arent stable, then we can just use `any`, but make sure to comment that once the types are ready, then we should add it